step-templates/network-add-ssl-certificate-binding.json

```diff
@@ -0,0 +1,52 @@
+{
+  "Id": "5a4857cf-dddc-4a08-a32e-8dfe018d986a",
+  "Name": "Network - Add SSL Certificate Binding",
+  "Description": "Creates an SSL certificate binding to a specific hostname and port using NETSH.",
```

Comment:
It would be good to mention that this will remove any cert that is configured for the _Hostname and Port_ variable, just to make sure that other users are aware.